app/proxy_info.py

```diff
@@ -9,71 +11,70 @@
     __name__, cfg.LOG_LEVEL_TABLE.get(__name__, cfg.DEFAULT_LOG_LEVEL)
 )
 
-# Example proxy_info.yaml(for subecu)
-"""
-# gateway: false
-enable_ota_proxy: true
-local_server: ["0.0.0.0", 8082]
-upper_ota_proxy: "http://10.0.0.2:8082"
-"""
-
-
 ###### Example proxy_info.yaml for different scenario ######
 ##                       mainecu                          ##
 # for mainecu, proxy_info.yaml is not needed, the following
 # DEFAULT_PROXY_INFO will be used.
 # we should treat the ecu with no proxy_info.yaml
 # as main ecu(as gateway), and enable ota_proxy without upper proxy.
-DEFUALT_PROXY_INFO = """
-enable_ota_proxy: true
+"""
+enable_local_ota_proxy: true
 gateway: true
+enable_local_ota_proxy_cache: true
```

Comment:
local_ota_proxy_enable_cache or enable_local_ota_proxy_cache?

Reply:
I think `enable_local_ota_proxy_cache` is better. Since we are using `enable_local_ota_proxy`.